x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_type_menu.tsx

```diff
@@ -20,14 +19,14 @@ import { SectionLoading } from '../../components/section_loading';
 
 interface Props {
   onActionTypeChange: (actionType: ActionType) => void;
-  actionTypes?: ActionType[];
+  featureId?: string;
```

Comment:
This is the connector type selection menu that is used on the Add Connector flyout. We want to allow all connector types to be created in the Stack connector management page, not just feature specific ones, so passing a feature id is optional here.